mods/common/languages/en.ftl

```diff
@@ -172,5 +283,289 @@ none = None
+
+## MapChooserLogic
+all-maps = All Maps
+no-matches = No matches
+player-players =
+    { $players ->
+        [one] { $players } Player
+       *[other] { $players } Players
+    }
+map-size-huge = (Huge)
+map-size-large = (Large)
+map-size-medium = (Medium)
+map-size-small = (Small)
+
+map-deletion-failed = Failed to delete map '{ $map }'. See the debug.log file for details.
+
+delete-map-title = Delete map
+delete-map-prompt = Delete the map '{ $title }'?
+delete-map-accept = Delete
+
+delete-all-maps-title = Delete maps
+delete-all-maps-prompt = Delete all maps on this page?
+delete-all-maps-accept = Delete
+
+## MissionBrowserLogic
+no-video-title = Video not installed
+no-video-text = The game videos can be installed from the
+    "Manage Content" menu in the mod chooser.
+no-video-cancel = Back
+
+cant-play-title = Unable to play video
+cant-play-prompt = Something went wrong during video playback.
+cant-play-cancel = Back
+
+## MusicPlayerLogic
+sound-muted = Audio has been muted in settings.
+no-song-playing = No song playing
+
+## MuteHotkeyLogic
+audio-muted = Audio muted.
+audio-unmuted = Audio unmuted.
+
+## PlayerProfileLogic
+loading-player-profile = Loading player profile...
+loading-player-profile-failed = Failed to load player profile.
+
+## ReplayBrowserLogic
+duration = Duration: { $time }
+singleplayer = Singleplayer
+multiplayer = Multiplayer
+
+today = Today
+last-week = Last 7 days
+last-fortnight = Last 14 days
+last-month = Last 30 days
+
+replay-duration-very-short = Under 5 min
+replay-duration-short = Short (10 min)
+replay-duration-medium = Medium (30 min)
+replay-duration-long = Long (60+ min)
+
+rename-replay-title = Rename Replay
+rename-replay-prompt = Enter a new file name:
+rename-replay-accept = Rename
+
+delete-replay-title = Delete selected replay?
+delete-replay-prompt = Delete replay { $replay }?
+delete-replay-accept = Delete
+
+delete-all-replays-title = Delete all selected replays?
+delete-all-replays-prompt =
+    { $count ->
+        [one] Delete { $count } replay.
+       *[other] Delete { $count } replays.
+    }
+delete-all-replays-accept = Delete All
+
+replay-deletion-failed = Failed to delete replay file '{ $file }'. See the debug.log file for details.
+
+## ReplayUtils
+incompatible-replay-title = Incompatible Replay
+incompatible-replay-prompt = Replay metadata could not be read.
+-incompatible-replay-recorded = It was recorded with an
+incompatible-replay-unknown-version = { -incompatible-replay-recorded } unknown version: { $version }.
+incompatible-replay-unknown-mod = { -incompatible-replay-recorded } unknown mod: { $mod }.
+incompatible-replay-unavailable-mod = { -incompatible-replay-recorded } unavailable mod: { $mod }.
+incompatible-replay-incompatible-version = { -incompatible-replay-recorded } incompatible version: { $version }.
+incompatible-replay-unavailable-map = { -incompatible-replay-recorded } unavailable map: { $map }.
```

Comment:
Nit: the indefinite article "an" belongs with the part of the sentence it's describing:
```ftl
-incompatible-replay-recorded = It was recorded with
incompatible-replay-unknown-version = { -incompatible-replay-recorded } an unknown version: { $version }.
```

Reply:
Addressed in #20207

Reply:
Yes, I thought so too, but then again you see the redundancy. I think this is really a matter of taste at this point.

Reply:
I'd advocate for even more redundancy in translations - we just can't be sure how different languages work and this split might actually be useless for a given language. It's also hiding context - the translator has to find the first part of the message to understand what is meant. In any case it's not a blocker for this PR. See also https://github.com/projectfluent/fluent/wiki/Good-Practices-for-Developers#prefer-wet-over-dry